README.md

```diff
@@ -7,20 +7,21 @@ the cursor and you will see the PDF file updated). Currently,
 vim-latex-live-preview only support UNIX-like systems. [Please let me know if
 you have any suggestions.](.github/CONTRIBUTING.md)
 
-Table of Contents
------------------
+![Screenshot with Evince](misc/screenshot-evince.gif)
```

Comment:
[Comment](https://github.com/xuhdev/vim-latex-live-preview/pull/39/files#diff-04c6e90faac2675aa89e2176d2eec7d8R177) should be moved or deleted.